swagger_server/controllers/itm_ta2_eval_controller.py

```diff
@@ -29,7 +31,13 @@ def get_alignment_target(session_id, scenario_id):  # noqa: E501
 
     :rtype: AlignmentTarget
     """
-    return ITM_SESSION.get_alignment_target(session_id=session_id, scenario_id=scenario_id)
+    session_dict = session_mapping.get(session_id)
+    adm_name = session_dict.get("adm_name") if session_dict else None
+    if not adm_name:
+        return 'Invalid Session ID', 400
+    else: # update access time
+        session_mapping[session_id] = {"adm_name": adm_name, "last_accessed": time.time()}
```

Comment:
Lines 34-39 are duplicated 4 times in the code below, I am wondering if there was any way to make this into a function to reduce code duplication?

Reply:
Yep, I remember noticing that too, not seeing anything obvious/immediate, but wanting to revisit it.  I'll take a look.

Reply:
Thank you for having me revisit that-- I've pushed an update.